src/components/tasks/TaskSearch/Suggestion/Options.tsx

```diff
@@ -0,0 +1,37 @@
+import { TaskSearchOption } from '@/interfaces/searchOptions.type';
+import className from './option.module.scss';
+import Option from './Option';
+interface IProp {
+    suggestions: Array<TaskSearchOption>;
+    activeSuggestionIndex: number;
+    onSuggestionSelected: (idx: number) => void;
+}
+export default function Options({
+    suggestions,
+    activeSuggestionIndex,
+    onSuggestionSelected,
+}: IProp) {
+    return (
+        <div className={className['suggestion-box']}>
```

Comment:
The styles variable name `className` can be different, that way it is less confusing with className attribute.

Reply:
We are using `className` throughout the code base. I think changing it will create a discrepancy